test/testcafe/spec/AuthenticatorYubikeyView_spec.js

```diff
@@ -100,13 +100,13 @@ test
 
     await t.expect(pageObject.getFormTitle()).eql('Verify with YubiKey');
     
-    await pageObject.submit();
+    await pageObject.clickVerifyButton();
 
-    pageObject.form.waitForErrorBox();
+    await pageObject.form.waitForErrorBox();
     await t.expect(pageObject.form.getErrorBoxText()).eql('We found some errors. Please review the form and make corrections.');
```

Comment:
These two are actually redundant assertions, now I see why the change was made in the Base form object. We could just use the waitForErrorBox assertion since it uses the name of the alert. We can update this later.

Reply:
Agreed, redundant